src/nxdoc/nuxeo-add-ons/nuxeo-for-adobe-cc.md

```diff
@@ -324,16 +336,21 @@ Using the Nuxeo CC Connector includes features within the traditional InDesign L
 --}}
 ![](nx_asset://8b19591e-f3a8-43fb-9fa6-04c3116200e2 ?w=350,border=true)
 
-Choose to work with the high resolution (HQ) or low resolution (LQ) rendition of the file by double clicking the entry in the Nuxeo column in the panel (you might need to expose the column in the “Panel Options” seen in the panel fly out).
+Choose to work with the high resolution (HQ) or low resolution (LQ) rendition of the file by double clicking the entry in the Nuxeo column in the panel (you might need to expose the column in the “Panel Options” seen in the links panel fly out).
 Be notified when updates have been made to the placed repository items, so that you can update the items in the InDesign document (note - you also see this on the placed image within the document)
 Select a link and choose to “Reveal in Nuxeo” which will open the file in your Nuxeo web instance to see more information about the file.
 
+Using the Nuxeo for Adobe CC Connector for creating and placing content has the added benefit of capturing incredibly useful information.  When you use repository content and link your assets and your InDesign files and then save the InDesign document to the repository,  behind the scenes, Nuxeo captures the id's of the placed assets and saves them as metadata in the repository for the InDesign file (as part of a compound doc schema).  The information is not exposed by default, but can be used by your Nuxeo team to create searches and views that show where content is used as part of other files.
+
+We are working on a sample to show you how this can be used in real life.  Check back here for updates.
+
+
 #### Working in Photoshop/Illustrator
 
-Photoshop and Illustrator have both access to Nuxeo CC Connector when you install the program.
+Photoshop and Illustrator have access to Nuxeo CC Connector when you install the program.
 
 If you want to edit an existing file from your Nuxeo account:
-1. Drag and Drop your asset from Connector, just like you would in InDesign.</br>
+1. Drag and Drop your asset from the connector.</br>
```

Comment:
The references to "the connector" in the previous sections have this word capitalised.